linetools/spectra/convolve.py

```diff
@@ -1,14 +1,16 @@
 """ Functions related to convolution.
       Taken from Barak by JXP
       May replace with scipy functions
+      NT: replace barak routines with astropy.convolvolution equivalents
 """ 
 
 # py2.6+ compatibility
 from __future__ import print_function, absolute_import, division, unicode_literals
 
 import numpy as np
+from astropy.convolution import convolve, Gaussian1DKernel, CustomKernel
 
-def convolve_psf(a, fwhm, edge='invert', replace_nan=False, debug=False):
+def convolve_psf_old(a, fwhm, edge='invert', replace_nan=False, debug=False):
```

Comment:
I think it's better to remove the old versions completely (`convolve_psf_old` and `convolve_window_old`).